### PR TITLE
updates RNSDK to 4.11.0

### PR DESCRIPTION
**Pull request by cedancp-cs**

### CODE REVIEW

**Comment by AldinaSculac** [comment]
Do you think we should `add` an example for a new way of implementing webviews? Also, I think you need to run `pod install`

**Review by big-toni — APPROVED** [review]
LGTM

**Comment by big-toni** [comment]
> @cedancp-cs Do you think we should `add` an example for a new way of implementing webviews? Also, I think you need to run `pod install`
> 
> @big-toni Should we add an example for the crash reporter?

Yes there should be an example for crash reporting. Maybe is better to do it in new PR?

**Comment by cedancp-cs** [comment]
> @cedancp-cs Do you think we should `add` an example for a new way of implementing webviews? Also, I think you need to run `pod install`
> 
> @big-toni Should we add an example for the crash reporter?

I updated it to include webview implementation and also updated pods.
Also I created a ticket for including crash reporter

File: ClassicImplementation/src/modules/Webviews/views/Webviews.tsx [hunk 1]
```diff
@@ -1,21 +1,33 @@
...
-          />
-        );
-      }}
-    />
+    <View>
+      <CSWebView>
+        {/* Implementation 1: Webview as a child */}
+        <WebView
+          source={{
+            uri: 'test.com',
```
Comment (AldinaSculac) [inline ClassicImplementation/src/modules/Webviews/views/Webviews.tsx:13]: Should we put here some real URL?

**Review by AldinaSculac — APPROVED** [review]

**Comment by cedancp-cs** [comment]
> LGTM! I'm just not sure in webview example if we should provide a real URL

hmm I don't think it makes too much difference or does it?

**Pull request merged**